Sources/SecureXPC/Server/XPCServer.swift

```diff
@@ -341,12 +353,56 @@ extension XPCServer {
         try XPCServiceServer._forThisXPCService()
     }
     
-    /// Creates a new anonymous server that only accepts connections from the same process it's running in.
-    internal static func makeAnonymous() -> XPCServer & NonBlockingServer {
+    /// Creates a new anonymous server that accepts requests from the same process it's running in.
+    ///
+    /// Only a client created from an anonymous server's endpoint can communicate with that server. Retrieve the ``NonBlockingServer/endpoint`` and
+    /// then create a client with it:
+    /// ```swift
+    /// let server = XPCServer.makeAnonymous()
+    /// let client = XPCClient.fromEndpoint(server.endpoint)
+    /// ```
+    ///
+    /// > Important: No requests will be processed until ``NonBlockingServer/start()`` or ``startAndBlock()`` is called.
```

Comment:
```
    /// - Important:
```

Reply:
Same note about this being the DocC format.